plugins/catalog/src/hooks/useEntity.ts

```diff
@@ -47,12 +47,13 @@ export const useEntityFromUrl = (): EntityLoadingStatus => {
   );
 
   useEffect(() => {
-    if (error || (!loading && !entity)) {
-      errorApi.post(new Error('Entity not found!'));
-      setTimeout(() => {
-        navigate('/');
-      }, REDIRECT_DELAY);
-    }
+    // Commenting to check EntityNotFound page
+    // if (error || (!loading && !entity)) {
```

Comment:
@shmidt-i Can this code block be removed now that we are not redirecting to /catalog when entity isn't found?

Reply:
Yes, please remove them and then we can merge!